.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "CodeGPT.apiKey": "Perplexity"
+}
```

Comment:
Does this belong here? (or to the personal one that isn't committed with the code?)